recipes/semver200/all/conandata.yml

```
@@ -0,0 +1,14 @@
+sources:
+  "1.1.0":
+    url: "https://github.com/zmarko/semver/archive/refs/tags/1.1.0.tar.gz"
+    sha256: bbea7e679684331b1059b773786559d790dd7aaa1e5ca24c0de12c9d5a4f2aed
+patches:
+  "1.1.0":
+    - patch_file: "patches/0001-unused-var-fix.patch"
+      patch_description: "Avoids: error: unused parameter 's' [-Werror,-Wunused-parameter]"
+      patch_type: "backport"
+      sha256: "ea7b241e8f37295e7943977fc66eaa6fa85d647b73c8cc1f4b566fd8078b0897"
+    - patch_file: "patches/0002-macro-invocation-fix.patch"
+      patch_description: "Fix minor() and major() macros invocation"
+      patch_type: "backport"
```

This one is marked backport but I did not find where it came from... perhaps?

```suggestion
      patch_type: "portability"
```

if this is indeed a backport it would be really nice to track the source